tests/test-team/template-mgmt-component-tests/nhs-app/template-mgmt-preview-nhs-app-page.component.ts

```diff
@@ -0,0 +1,185 @@
+import { test, expect } from '@playwright/test';
+import SessionStorageHelper from '../../helpers/session-storage-helper';
+import { TemplateMgmtPreviewNhsAppPage } from '../../pages/nhs-app/template-mgmt-preview-nhs-app-page';
+import { SessionFactory } from '../../helpers/session-factory';
+import {
+  assertFooterLinks,
+  assertGoBackLink,
+  assertLoginLink,
+  assertNotifyBannerLink,
+  assertSkipToMainContent,
+} from '../template-mgmt-common.steps';
+
+const sessions = {
+  empty: SessionFactory.createNhsAppSession('empty-nhs-app-preview-session'),
+  valid: {
+    ...SessionFactory.createNhsAppSession('valid-nhs-app-preview-session'),
+    nhsAppTemplateName: 'test-template-nhs-app',
+    nhsAppTemplateMessage: 'test-template-message',
+  },
+};
+
+test.describe('Preview NHS App template Page', () => {
+  const sessionStorageHelper = new SessionStorageHelper(
+    Object.values(sessions)
+  );
+
+  test.beforeAll(async () => {
+    await sessionStorageHelper.seedSessionData();
+  });
+
+  test.afterAll(async () => {
+    await sessionStorageHelper.deleteSessionData();
+  });
+
+  test('when user visits page, then page is loaded', async ({
+    page,
+    baseURL,
+  }) => {
+    const previewNhsAppTemplatePage = new TemplateMgmtPreviewNhsAppPage(page);
+
+    await previewNhsAppTemplatePage.loadPage(sessions.valid.id);
+
+    await expect(page).toHaveURL(
+      `${baseURL}/templates/preview-nhs-app-template/${sessions.valid.id}`
+    );
+
+    await expect(previewNhsAppTemplatePage.editRadioOption).not.toBeChecked();
+
+    await expect(previewNhsAppTemplatePage.submitRadioOption).not.toBeChecked();
+
+    await expect(previewNhsAppTemplatePage.pageHeader).toContainText(
+      'NHS App message template'
+    );
+
+    await expect(previewNhsAppTemplatePage.pageHeader).toContainText(
+      'test-template-nhs-app'
+    );
+
+    await expect(previewNhsAppTemplatePage.messageText).toHaveText(
+      'test-template-message'
+    );
+  });
+
+  test.describe('Page functionality', () => {
```

Comment:
I think adding in a test for the character count would be worth it (I think SMS has one)

Reply:
I think that is in the create page, I will add a create page test and add it to it

Reply:
Done